dev/archery/archery/utils/lint.py

```diff
@@ -90,20 +91,75 @@ def cpp_linter(src, build_dir, clang_format=True, cpplint=True,
 
 
 class CMakeFormat(Command):
-    def __init__(self, cmake_format_bin):
-        self.bin = cmake_format_bin
+
+    # TODO(kszucs): check cmake_format version
```

Comment:
I don't know if you can do it in this PR?

Reply:
Wait, it seems done already below. Perhaps just remove this TODO?

Reply:
Yep, thanks for spotting that!